src/components/common/YourCompany/EntityName.vue

```diff
@@ -52,7 +52,8 @@
               <span class="info-text">{{ GetCorpFullDescription(getEntityType) }}</span>
             </div>
             <div class="info-text pt-3">
-              <span>The name of this business will be the current Incorporation Number followed by "B.C. Ltd."</span>
+              <span>The name of this business will be the current Incorporation Number followed
+                by "B.C. {{ getUpdatedName() }}"</span>
             </div>
```

Comment:
fixed the error: change Named business to Numbered -> showing wrong numbered name, always showed "B.C. Ltd".